packages/core/docs/performance/other-optimizations.md

```diff
@@ -0,0 +1,123 @@
+
+## Swap fonts :ledger:
+
+Leverage the `font-display` CSS feature to ensure that the text is visible to the user while web fonts are loading.
+
+```css
+@font-face {
+  font-family: 'Arial';
+  font-display: swap;
+}
+```
+
+The `font-display` API specifies how a font is displayed. The `swap` value tells the browser that the text using this font should be displayed immediately using a system font. Once the custom font is ready, it replaces the system font.
+
+If you are using Google Fonts, add the `&display=swap` parameter to the end to the Google Fonts URL:
+
+```html
+<link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
+```
+
+If you are using the `@nuxtjs/google-fonts` package, you can use the [display](https://google-fonts.nuxtjs.org/options#display) property.
+
+## Avoid using more than one library version per page :ledger:
+
+Check if your website doesn't use multiple versions of the same library. This will make the user download unnecessary data and slow down your page. Clean up the code and make sure you only use one version by running the following command. It will generate a report of what's in the final bundle.
+
+```bash
+yarn nuxt build --analyze
+```
+
+## Compress text content :ledger:
+
+Ensure you compress HTML, JSON, JavaScript, CSS, and SVG. It will save bandwidth for you and the user and make the page load faster.
+
+There are two ways you can do that:
+
+* (Recommended) Enable GZIP or Brotli compression on the server.
+* To enable compression in Nuxt.js using the [nuxt compression](https://github.com/nuxt-modules/compression) or [nuxt-compress](https://www.npmjs.com/package/nuxt-compress) modules.
+
+## Don't close connection used multiple times :blue_book:
+
+Use `keep-alive` headers and don't close the connection when multiple requests are sent to the same domain.
+
+By default, when user enters your website, browser needs to create new connection for every resource request (JS, CSS, images etc). This requires performing three-way handshake every time. 
+
+Using keep-alive helps us reduce this process to minimum, improving our round trip time (RTT) and page load time.
+
+* The HTTP/1.0 clients request persistent connection by sending the `Connection: keep-alive` request header to the server.
+* In HTTP/1.1, all connections are considered persistent unless declared otherwise.
+* In HTTP/2 connection and Keep-Alive headers are not allowed. Chrome and Firefox ignore them, but Safari will not load the web page if the server returns these headers in error.
+
+```javascript
+Example:
+GET /echo HTTP/1.0
+Host: vuestorefront.io
+Connection: keep-alive
+Keep-Alive: timeout=5, max=100
+```
+
+## Preconnect with external domains like your CDN or fonts provider :ledger:
```

Comment:
needs better explaination as too many preconnects can also lead to worsen perf, be more specific, give a warning

Reply:
Fixed